js/data/create_bucket.js

```diff
@@ -15,6 +16,10 @@ function createBucket(layer, buffers, collision, indices) {
         return;
     }
 
+    for (var k in layer.layout) {
+        layer.layout[k] = new StyleDeclaration('layout', layer.type, k, layer.layout[k]).value;
```

Comment:
Hm, no, even when I change those z-levels to 10 and 13, they never recalculate on zoom using this method…